packages/json/src/browser/json-client-contribution.ts

```diff
@@ -67,7 +67,7 @@ export class JsonClientContribution extends BaseLanguageClientContribution {
         for (const s of allConfigs) {
             if (s.fileMatch) {
                 for (let fileMatch of s.fileMatch) {
-                    if (fileMatch.charAt(0) !== '/' && !fileMatch.match(/\w+:\/\//)) {
+                    if (fileMatch.charAt(0) !== '/' && !fileMatch.match(/\w+:/)) {
```

Comment:
@svenefftinge would be fine to remove them?

Reply:
It's fine for now but might pop up again, once we use the vs code extension.